charts/vela-core/templates/defwithtemplate/autoscale.yaml

```
@@ -0,0 +1,50 @@
+apiVersion: core.oam.dev/v1alpha2
+kind: TraitDefinition
+metadata:
+  name: autoscale
+  annotations:
+    definition.oam.dev/apiVersion: standard.oam.dev/v1alpha1
+    definition.oam.dev/kind: Autoscaler
+    definition.oam.dev/description: "Automatically scale workloads by cron or resource utilization"
+spec:
+  appliesToWorkloads:
+    - webservice
+    - backend
+    - deployments.apps
+    - podspecworkload
+  workloadRefPath: spec.workloadRef
+  definitionRef:
+    name: autoscalers.standard.oam.dev
+  extension:
+    template: |
+      output: {
+          apiVersion: "standard.oam.dev/v1alpha1"
+          kind:       "Autoscaler"
+          spec: {
+              minReplicas: parameter.minReplicas
+              maxReplicas: parameter.maxReplicas
+              triggers: [{
+                  name: parameter.name
+                  type: parameter.type
+                  condition: {
+                      startAt:  parameter.startAt
+                      duration: parameter.duration
+                      days:     parameter.days
+                      replicas: parameter.replicas
+                      timezone: parameter.timezone
+                  }
+              }, ...]
+          }
+        }
+      parameter: {
+      	minReplicas: *1 | int
+      	maxReplicas: *4 | int
+      	name:        *"" | string
+      	type:        *"cron" | string
```

```suggestion
      	type:        "cron"
```

I suggest we naming this traitdefinition by `cronscale` and directly write `cron` as type here.